src/components/Explore/queries/generateMetricsQuery.ts

```diff
@@ -1,13 +1,15 @@
+import { getTraceExplorationScene } from 'utils/utils';
 import { ALL, MetricFunction, VAR_FILTERS_EXPR, VAR_DURATION_PERCENTILES_EXPR } from '../../../utils/shared';
+import { SceneObject } from '@grafana/scenes';
 
 interface QueryOptions {
   metric: MetricFunction;
   extraFilters?: string;
   groupByKey?: string;
-  sample?: boolean;
+  sample?: string;
```

Comment:
Not a fan of having the sample be a free string, there's more potential for mistakes when generating the query. If we must support different sample parameters can you consider an enum? Then we can control what get's added to the query for each enum value in the generateMetricsQuery function.